docs/_components.md

```diff
@@ -85,9 +85,9 @@ const App = () => (
 ### Show Tabs with Source
 <Tabs>
   <TabItem value="ts" label="TS" default>
-```tsx title="<file />Bitterson/help.ts" link="https://figma.com"
+```tsx title="<file />Bittensor/help.ts" link="https://figma.com"
```

Comment:
This entire document is hidden, unpublished, by design. This was an experiemental thing back when we were coding up the site elements. Not reviewing rest of this doc.